test/integration/quoted_and_unquoted_searches_test.rb

```diff
@@ -0,0 +1,178 @@
+require "integration_test_helper"
+
+class QuotedAndUnquotedSearchTest < IntegrationTest
+  def setup
+    # `@@registries` are set in Rummager and is *not* reset between tests. To
+    # prevent caching issues we manually clear them here to make a "new" app.
+    Rummager.class_variable_set(:'@@registries', nil)
+
+    stub_elasticsearch_settings
+    create_meta_indexes
+    reset_content_indexes
+    end
+
+  def teardown
+    clean_test_indexes
+  end
+
+
+  # NEW WEIGHTING TESTS
```

Comment:
@jackscotti Yes, normally :thumbsup:. But the "new weighting" here refers to the new search weighting that was implemented a few months back but is still under the feature flag `&debug=new_weighting`, so it's okay here.